src/queries.js

```diff
@@ -1,4 +1,6 @@
 import {matches} from './matches'
+import prettyFormat from 'pretty-format' // eslint-disable-line
```

Comment:
Why eslint-disable-line here?

Reply:
So basically `pretty-format` is not part of the package.json, dependencies but its part of `jest` I believe. Hence I have added the `eslint-disable-line`, yes, I need to clean up a bit by adding the exact eslint-ignore.

Reply:
But what if I don't use jest with dom-testing-library?

Reply:
Sorry, `pretty-format` is from `jest-matcher-utils` (https://github.com/facebook/jest/blob/master/packages/jest-matcher-utils/package.json#L14) which is our dependencies. Sorry for the rushed comment, saying it as `jest`!

Reply:
I'm not against using pretty-format, I'm just trying to understand if there is a reason to not put it in package.json as a direct dependency. Then there is no need to fight around eslint complaining (rightly). And if jest-matcher-utils switch to something else internally and stop requiring pretty-format, our package will still work.

Reply:
Agreed

Reply:
@lgandecki Yes, I agree. Will make the changes. Thanks your comments!